tests/helper/helper_filesystem.go

```diff
@@ -247,3 +247,29 @@ func ReplaceDevfileField(devfileLocation, field, newValue string) error {
 	}
 	return nil
 }
+
+// CopyFile copies the file from source to destination
```

Comment:
Add more details. For example - what is source and destination argument

Reply:
> Add more details. For example - what is source and destination argument

I think specifying the source and destination directory will conserve the ```CopyFile``` function for specific case. But it should be global so that anyone can use this function anywhere. So this function basically copies the files present in source directory to the destination directory. IMO comment explain it more accurately. Let me know if you still feel we should some comment.

Reply:
Can please check how https://github.com/openshift/odo/blob/master/tests/helper/helper_filesystem.go#L144 has been implemented in other places. I am sure this will gives you a better picture and instead of writing the function again just re use it. See how it has been done in - https://github.com/openshift/odo/blob/master/tests/helper/helper_filesystem.go#L92

Reply:
Ahh, I just missed it. Thanks for the info, I will definitely use this 👍